bundleSize.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs');
+const path = require('path');
+
+const newDistPath = './dist';
+const oldDistPath = './dist_old';
+
+const getFileSizes = folderPath => {
+  return fs.readdirSync(folderPath).reduce((res, fileName) => {
+    res[fileName] = fs.readFileSync(path.join(folderPath, fileName)).byteLength;
+    return res;
+  }, {});
+};
+
+const oldValues = getFileSizes(oldDistPath);
+const newValues = getFileSizes(newDistPath);
+
+Object.entries(oldValues).forEach(([name, size]) => {
+  const newSize = newValues[name];
+  const delta = newSize - size;
+  const sizeColorFg = delta <= 0 ? '\x1b[32m' : '\x1b[31m';
+  const reset = '\x1b[0m';
```

Comment:
I spent some minutes try to figure out what this reset variable stood for, It's maybe only because I'm not used to manipulate colors in console outputs but maybe a naming like "resetColorFg" or something like this would helped me a bit.

Reply:
sure I can rename it